formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/internal/ProtobufEncoding.kt

```diff
@@ -147,6 +150,24 @@ internal open class ProtobufEncoder(
         }
     }
 
+    private fun <T> encodeOneOfValue(serializer: SerializationStrategy<T>, value: T) {
+        if (serializer is SealedClassSerializer) {
```

Comment:
Again, you can (maybe should) support any kind of polymorphic child (you can extract the information from the serializersModule).